libvast/test/type.cpp

```diff
@@ -671,7 +668,7 @@ TEST(hashable) {
   CHECK_NOT_EQUAL(hash(type{bool_type{}}), hash(type{address_type{}}));
   auto x = record_type{
     {"x", integer_type{}}, {"y", string_type{}}, {"z", list_type{real_type{}}}};
-  CHECK_EQUAL(hash(x), 7434606127152318469ul);
```

Comment:
Was there a way not to break here by adding a stub type into the type variant? The hash only changes because the indices after port moved one up.

Reply:
To clarify, this whole thing is mostly a breaking change because the hashes changed.

Reply:
Unfortunately there's no way around this change. You nailed the issue.